Dockerfile-3.8

```diff
@@ -2,12 +2,21 @@
 
 FROM ubuntu:20.04
 
-ENV VIRTUAL_ENV=/blockstore/venv
-
 RUN apt-get update
-RUN apt-get install libmysqlclient-dev libjpeg-dev libssl-dev libffi-dev python3 python3-venv python3-pip git -y
```

Comment:
`libssl-dev` and `libffi-dev` were required by the `cryptography` package but according to the [docs](https://cryptography.io/en/latest/installation/?highlight=libffi#building-cryptography-on-linux), with the latest version of pip, a binary wheel is installed so those are no longer needed.